### PR TITLE
NODE-141: The REST endpoint /node/version contains the commitId

### PR DESCRIPTION
**Pull request by vsuharnikov**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -4,17 +4,15 @@ resolvers ++= Seq(
...
-addSbtPlugin("se.marcuslonnberg" % "sbt-docker" % "1.4.1")
+Seq(
+  "com.eed3si9n" % "sbt-assembly" % "0.14.3",
+  "com.typesafe.sbt" % "sbt-native-packager" % "1.2.0-M8",
+  "org.scalastyle" %% "scalastyle-sbt-plugin" % "0.8.0",
+  "org.scoverage" % "sbt-scoverage" % "1.5.0",
+  "com.github.gseitz" % "sbt-release" % "1.0.3",
+  "se.marcuslonnberg" % "sbt-docker" % "1.4.1",
+  "com.typesafe.sbt" % "sbt-git" % "0.9.3"
+).map(addSbtPlugin)
```
Comment (phearnot) [inline project/plugins.sbt]: Shouldn't this be `forEach`?

Comment (phearnot) [reply project/plugins.sbt]: Well okay, we do need `Seq[Setting[_]]` here.

**Comment by vsuharnikov** [comment]
From SBT:
```
> reload
…
> show version
[info] 0.7.5-216-g7a1a958
> compile
```
Version.scala is:
```scala
package com.wavesplatform

object Version {
  val VersionString = "0.7.5"
  val VersionTuple = (0, 7, 5)
  val RawVersionString = "0.7.5-216-g7a1a958"
}
```

**Comment by vsuharnikov** [comment]
Well, the last fix:
```
> show version
[info] 0.7.5-216-g7a1a958-SNAPSHOT
```
```scala
package com.wavesplatform

object Version {
  val VersionString = "0.7.5-216-g7a1a958-SNAPSHOT"
  val VersionTuple = (0, 7, 5)
}
```

**Comment by alexeykiselev** [comment]
Why do you have SNAPSHOT suffix in your version example. I've tried and I see a normal long version w/o SNAPSHOT. How come?!

**Comment by alexeykiselev** [comment]
I've realised that it was uncommitted changes signifier. I've replaced SNAPSHOT with DIRTY, as in Git itself.

**Pull request merged**